src/js/gi/selectors/calculator.js

```diff
@@ -0,0 +1,23 @@
+import { createSelector } from 'reselect';
+
+const getConstants = (state) => state.constants.constants;
+
+const getEligibilityDetails = (state) => {
+  const details = Object.assign({}, state.eligibility);
+  delete details.dropdowns;
```

Comment:
`details.dropdowns` holds the shown/not-shown state of the eligibility interdependent dropdowns and isn't used in the calculation. IOW the code for the visible/hidden presentation of the eligibility dropdowns is separate from the code that determines what chosen values become inputs to the calculation.